scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
```

Comment:
The change from `base64 -w 0` to `cat | base64 | tr -d '
'` is less efficient as it uses more processes and pipes. Consider keeping the original `base64 -w 0` which is more direct and performant.